iati/about/templates/about/case_study_index_page.html

```diff
@@ -0,0 +1,68 @@
+{% extends "base.html" %}
+{% load wagtailcore_tags wagtailimages_tags wagtail_modeltranslation %}
+
+{% block body_class %}body body--news-single{% endblock %}
```

Comment:
Comment for possible refactor
Suggest we set these as `{% set body_class = 'body body--news-single' %}` - though this may need to be set before extending `base.html` - to be explored